hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/client/functional/TestHoodieFileSystemViews.java

```diff
@@ -161,11 +160,7 @@ public void testFileSystemViewConsistency(HoodieTableType tableType, boolean ena
       // mimic failed write for last completed operation and retry few more operations.
       HoodieInstant lastInstant = metaClient.reloadActiveTimeline().getWriteTimeline().lastInstant().get();
       HoodieCommitMetadata commitMetadata = metaClient.getActiveTimeline().readCommitMetadata(lastInstant);
-      StoragePath instantPath = HoodieTestUtils
-          .getCompleteInstantPath(metaClient.getStorage(),
-              metaClient.getTimelinePath(),
-              lastInstant.requestedTime(), lastInstant.getAction(), HoodieTableVersion.fromVersionCode(writeVersion));
-      metaClient.getStorage().deleteFile(instantPath);
+      client.rollback(lastInstant.requestedTime());
```

Comment:
the comment says "// mimic failed write for last completed operation and retry few more operations", does it make sense to rollback the instant directly?